shared/src/components/Header/HardwareMenu/HardwareMenu.test.tsx

```diff
@@ -1,17 +1,24 @@
-import { shallow } from "enzyme";
+import { render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import React from "react";
 
 import { HardwareMenu } from "./HardwareMenu";
 
+const generateLink = (link, linkClass) => (
```

Comment:
I'm surprised TS isn't complaining about these not being typed. Not a big deal either way since we'll be moving all this into the `ui` workspace soon(ish) anyway.

Reply:
It doesn't need them to be typed because it can correctly infer the type based on how it's used. It probably still should be for developer documentation purposes :)